api/src/main/java/au/org/aodn/nrmn/restapi/validation/validators/row/formatted/MeasureBetweenL5l95.java

```diff
@@ -40,7 +40,7 @@ public MeasureBetweenL5l95() {
         if (measureJson.isEmpty() || (l5 == 0 && l95 == 0))
             return Validated.valid("No expected sizing");
 
-        boolean isInvertSized = target.getIsInvertSizing();
+        boolean isInvertSized = target.getIsInvertSizing() ? target.getIsInvertSizing() : false;
```

Comment:
I'm pretty sure this will throw a null pointer exception if getIsInvertSizing is null

Reply:
Actually throws this:

    incompatible types: <null> cannot be converted to boolean

Reply:
Will rework this check to be more robust.